fabrictestbed_extensions/fablib/fablib.py

```diff
@@ -1006,6 +1006,11 @@ def get_image_names(self) -> List[str]:
             "default_ubuntu_20",
             "default_ubuntu_21",
             "default_ubuntu_22",
+            "default_fedora_36",
+            "default_fedora_37",
+            "docker_rocky_8",
+            "docker_ubuntu_20",
+            "docker_ubuntu_22"
         ]
```

Comment:
Images to be added:
- default_debian_11
- default_rocky_9

Images to be removed:
- default_cirros
- default_freebsd_13_zfs
- default_openbsd_7